types/weixin-app/weixin-app-tests.ts

```diff
@@ -0,0 +1,3 @@
+import {} from 'weixin-app';
```

Comment:
As mentioned before, don't use an import for this. Users of the library should use `/// <reference types="weixin-app" />`, and for the test file you don't need the reference at all (since the definitions are already included).